it/it-tests/src/test/java/it/organization/OrganizationMembershipTest.java

```diff
@@ -84,7 +84,6 @@ public void tearDown() throws Exception {
   @Test
   public void new_user_should_not_become_member_of_default_organization() throws Exception {
     String login = createUser();
-
```

Comment:
It's because I already tried to fix this test in a previous commit with the same message, but I missed some cases and we still had some FP.